external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -50,7 +50,7 @@ class DenseHyperparameters(Hyperparameters):
             each output variable.
         save_model_checkpoints: if True, save one model per epoch when
             dumping, under a 'model_checkpoints' subdirectory
-        clip_config: configuration of dataset packing.
+        clip_config: configuration of input and output clipping
```

Comment:
Can you add to this to mention it is always the last/feature dim?

Reply:
Updated.